### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Yash-Garg**

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ In the new frontier of DevOps it is becoming more and more popular to interact w
 
 ## Quick start
 
-```
+```python
```
Comment (aslafy-z) [inline README.md]: This is not python code but shell!

Comment (ssiddhantsharma) [reply README.md]: That's what intrigued me too!

Comment (jacobtomlinson) [reply README.md]: Ha I totally missed that. That'll teach me for doing such a speedy review!

Comment (jacobtomlinson) [reply README.md]: Fixed on master!

Comment (ssiddhantsharma) [reply README.md]: @jacobtomlison Be careful of documentation change PR's. Most of the times these are just to gain some swag or stickers and not really contribute to the true documentation.

I always stay away from PR's that change 4 lines  in documentation!

Comment (jacobtomlinson) [reply README.md]: Totally agree and I nearly rejected it on those grounds, however at the moment I am trying to spread the word about opsdroid. So I'm happy to send out the stickers. If it starts happening more often I will become more strict.

**Pull request merged**